app/assets/javascripts/uploadAdditionalInformation.js

```diff
@@ -81,6 +84,8 @@ $(document).ready(function(){
                 // Take upload URL and pass it into the second form
                 $("#additional_information_upload_additional_information").val($.parseXML(response).getElementsByTagName("Key")[0].childNodes[0].nodeValue);
                 $("#additional_information_upload_file_name").val(file.name);
+                // Re-enable submit button as upload has been successful
+                $(":submit").prop("disabled", false);
```

Comment:
Not sure if I'm reading this right, but isn't this before the file gets uploaded to S3 ? Should it not be after ?